manifests/rhoai/addon/params.env

```diff
@@ -1,5 +1,6 @@
 # Injected variables from the Operator
 odh-dashboard-image=quay.io/opendatahub/odh-dashboard:main
-oauth-proxy-image=registry.redhat.io/openshift4/ose-oauth-proxy-rhel9@sha256:ca21e218e26c46e3c63d926241846f8f307fd4a586cc4b04147da49af6018ef5
+kube-rbac-proxy=quay.io/opendatahub/odh-kube-auth-proxy@sha256:dcb09fbabd8811f0956ef612a0c9ddd5236804b9bd6548a0647d2b531c9d01b3
```

Comment:
shouldn't this one and the one for onprem be `rhoai`

Reply:
> shouldn't this one and the one for onprem be rhoai

That does not matter at all, rhoai images are replaced by the operator